Source/WebCore/page/EventHandler.cpp

```diff
@@ -3918,8 +3918,9 @@ void EventHandler::defaultKeyboardEventHandler(KeyboardEvent& event)
 
     if (event.type() == eventNames().keydownEvent) {
         m_frame.editor().handleKeyboardEvent(event);
-        if (event.defaultHandled())
+        if (event.defaultHandled() && !(event.keyIdentifier() == "PageDown"_s || event.keyIdentifier() == "PageUp"_s))
```

Comment:
Two lines down it checks `key()`, not `keyIdentifier()`. Is there a difference?

Reply:
I'm not sure, but for PageUp / PageDown checking, it uses `keyIdentifier`, so I believe that is the correct one to use for this case